apps/sms/test/marionette/lib/views/new-message/accessors.js

```diff
@@ -14,6 +14,9 @@ var SELECTORS = Object.freeze({
   subjectInput: '.subject-composer-input',
   sendButton: '#messages-send-button',
   attachButton: '#messages-attach-button',
+  headerActionButton: '#messages-options-button',
+  reportHeaderAction: '#information-report-header',
```

Comment:
These two fields should not be placed inside this file. This file is for selectors (CSS selectors) that are applicable for the New Message View only (the view where you edit SMS/MMS message).

"report" - is for the Message Report View;
"group" - is for the Message Group View;

Currently selectors for the "group" and "report" views are located inside shared ```messages.js``` file [1].

Please, find the appropriate place for those in this file and move it there (see other similar properties in this file). If something is unclear, please ask me.

[1] https://github.com/mozilla-b2g/gaia/blob/187a8e94f9edba20fc6260d66883ed82a8aef647/apps/sms/test/marionette/lib/messages.js